README.md

```diff
@@ -13,63 +13,68 @@ validate, edit, and export SCAP documents.
 
 ## Compilation
 
-Choose *1a* or *1b* depending on whether you want sources from a release tarball or the git repository.
+Choose *2a* or *2b* depending on whether you want sources from a release tarball or the git repository.
 
-1a) Get a release tarball
+1) The following packages are required to be installed on your system:
+```
+autoconf automake libtool gcc
+```
+
+2) a) Get a release tarball and run ./autogen.sh:
 ```
 # replace ${version} with the desired version
 wget https://github.com/OpenSCAP/openscap/releases/download/${version}/openscap-${version}.tar.gz
 tar -xzpf openscap-${version}.tar.gz
 cd openscap-${version}
+./autogen.sh
 ```
 
 **OR**
 
-1b) Get fresh sources from git repository and run ./autogen.sh
-
-The autoconf, automake, and libtool tools are required to be installed on your system.
+2) b) Get fresh sources from git repository and run ./autogen.sh:
 
 ```
 git clone https://github.com/OpenSCAP/openscap.git
 cd openscap
 ./autogen.sh
 ```
 
-2) Run the following commands to build the library:
+3) To build the library you need the following build dependencies
+(some of these are optional, if they are not detected, openscap will be compiled
+without respective optional features):
+```
+dbus-devel GConf2-devel libacl-devel libblkid-devel libcap-devel libcurl-devel \
+libgcrypt-devel libselinux-devel libxml2-devel libxslt-devel make openldap-devel \
+pcre-devel perl-XML-Parser perl-XML-XPath perl-devel python-devel rpm-devel swig \
+bzip2-devel
+```
+When you have all the build dependencies installed you can run the following
+commands to build the library:
 ```
 ./configure
 make
 ```
-Build dependencies:
-(some of these are optional, if they are not detected, openscap will be compiled
-without respective optional features)
- * dbus-devel
- * GConf2-devel
- * libacl-devel
- * libblkid-devel
- * libcap-devel
- * libcurl-devel
- * libgcrypt-devel
- * libselinux-devel
- * libxml2-devel
- * libxslt-devel
- * make
- * openldap-devel
- * pcre-devel
- * perl-XML-Parser
- * perl-XML-XPath
- * perl-devel
- * python-devel
- * rpm-devel
- * swig
- * bzip2-devel
-
-3) Run library self-checks by executing the following command:
+
+4) After building the library you might want to run library self-checks. To do
+that you need to have these additional packages installed:
+```
+wget lua which procps-ng initscripts chkconfig sendmail
+```
+and it is also required to have `sendmail` service running on the system:
+```
+systemctl start sendmail.service
+```
+Now you can execute the following command to run library self-checks:
 ```
 make check
 ```
+Note: If you want to run `make distcheck` you will also need to install
+`rubygems` package and install `asciidoctor`:
+```
+gem install asciidoctor
```

Comment:
There is a package `rubygem-asciidoctor` on my Fedora, so you don't have to run `gem`, you can run only dnf.